src/cursor.zig

```diff
@@ -1,8 +1,23 @@
 const std = @import("std");
-
+const testing = std.testing;
+const fixedBufferStream = std.io.fixedBufferStream;
 const esc = "\x1B";
 const csi = esc ++ "[";
 
+pub const CursorMode = enum(u8) {
+    BLINKING_BLOCK = 1,
```

Comment:
The Zig naming convention is that enum values should be camel case, so e.g. `blinking_block`. You can make an exception for the `I` beam (e.g. `blinking_I_beam`).